### DISCUSSION TITLE
What if I further my education from an institution that has GitHub Student Developer Pack, and I move to an institution that has GitHub Student Developer Pack? Do I have to reapply?

### DISCUSSION THREAD

**Discussion by SG3086X** [community/community#60105] [GitHub Education] [answered]
### Hi everyone,

Put your answer here.

**Comment by nirav-gajera** [comment, marked as answer]
If you move to an institution that also has access to the GitHub Student Developer Pack, you do not need to reapply for the pack. Once you have been approved for the GitHub Student Developer Pack, your access to the benefits will continue, regardless of the institution you are attending

However, it's essential to ensure that you are still eligible for the GitHub Student Developer Pack at your new institution. Some institutions might have different eligibility criteria or may require you to verify your student status again. Be sure to review the specific terms and conditions associated with the GitHub Student Developer Pack at your new institution to ensure you continue to comply with their requirements.
